azurerm/internal/services/network/tests/application_gateway_resource_test.go

```diff
@@ -2044,8 +2044,11 @@ resource "azurerm_web_application_firewall_policy" "testfwp" {
   location            = azurerm_resource_group.test.location
 
   policy_settings {
-    enabled = true
-    mode    = "Prevention"
+    enabled                     = true
+    mode                        = "Prevention"
+    file_upload_limit_in_mb     = 100
+    max_request_body_size_in_kb = 100
+    request_body_check          = "true"
```

Comment:
Again, I don't think these are necessary here.

Reply:
Removed the settings.